components/ingest-service/pipeline/message/chef_run.go

```diff
@@ -14,6 +14,14 @@ import (
 
 type ChefRunPipe func(<-chan ChefRun) <-chan ChefRun
 
+func PropogateChefRun(out chan<- ChefRun, msg *ChefRun) {
```

Comment:
We don't need to do it in this PR, but this might be a nice place for a small interface for Messages so we can share some functions across these pipelines.

Reply:
i tried this last week and failed. Gave up and decided it wouldn't be possible without generics. but if you have ideas, please share

Reply:
I meant something like this: https://github.com/chef/automate/commit/02d5ccb372e142ec9ff9682b005ca99e34243b25#diff-7056bfe5063a802fbe9dc89de9191f26R5-R8. But I admittedly didn't look at it past a surface level.  Was there something specific you hit?

Reply:
i think thats what i was trying to do. but you cant have a function that takes a `chan Finishable` and give it a `chan ChefRun`.

Reply:
The cure is worse than the disease I think. Carry on.